docs/src/pages/[platform]/getting-started/migration/migration.angular.mdx

```diff
@@ -215,3 +215,7 @@ please see the [Form Field Customization](../connected-components/authenticator/
 ### CSS Styling
 
 The 2.x version of the `Authenticator` has a completely different set of CSS variables. Please look over the [Amplify CSS Variables](../connected-components/authenticator/customization#styling) section for more information.
+
+### `onAuthUiStateChange`
```

Comment:
This should be 
```suggestion
### `onAuthUIStateChange`
```
(see legacy [code](https://github.com/aws-amplify/amplify-ui/blob/46cdfb1955fad14b2f7931f646fb6b18f4c39f6c/legacy/amplify-ui-components/src/common/helpers.ts#L90))